client/src/locales/en.json

```diff
@@ -100,6 +100,7 @@
   "user.auth.input.password.confirm": "Confirm password",
   "user.auth.input.new-password.confirm": "Confirm new password",
   "user.auth.input.password.invalid": "Please use min 4 and max 32 characters without spaces.",
+  "common.auth.input.wrong-password": "Please enter correct current password.",
```

Comment:
just `Please enter the correct password. `

current password label is displayed above the input